src/modules/esl-panel-group/core/esl-panel-group.ts

```diff
@@ -131,64 +117,80 @@ export class ESLPanelGroup extends ESLBaseElement {
     }
   }
 
-  /** @returns Panels that are processed by the current panel group */
+  /** @returns ESLMediaRuleList instance of the mode mapping */
+  @memoize()
+  public get modeRules() {
```

Comment:
return type...

Reply:
We have no restrictive rules on return type definition.
Let's discuss that point on ESL call and enable linting if we decide to always have return type to be defined explicitly.

Reply:
UPD: create a discussion placeholder for that point: #748